packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6643,27 +6642,28 @@ class RepaintBoundary extends SingleChildRenderObjectWidget {
 class IgnorePointer extends SingleChildRenderObjectWidget {
   /// Creates a widget that is invisible to hit testing.
   ///
-  /// The [ignoring] argument must not be null. If [ignoringSemantics] is null,
-  /// this render object will be ignored for semantics if [ignoring] is true.
+  /// The [ignoring] and [ignoringSemantics] arguments must not be null.
   const IgnorePointer({
     super.key,
     this.ignoring = true,
-    this.ignoringSemantics,
+    this.ignoringSemantics = false,
```

Comment:
I think the `ignoringSemantics` was a workaround to provide people a way to get the semanitcs back if ignoring = true, but I am not sure.

If so,  should we deprecate this and just ask developer to use ExcludeSemantics?